src/background.js

```diff
@@ -26,28 +26,36 @@ const MOBILENET_MODEL_TFHUB_URL =
 const IMAGE_SIZE = 224;
 // The minimum image size to consider classifying.  Below this limit the
 // extension will refuse to classify the image.
-const MIN_IMG_SIZE = 128;
+const MIN_IMG_SIZE = 32;
 
 // How many predictions to take.
-const TOPK_PREDICTIONS = 2;
+const TOPK_PREDICTIONS = 1;
 const FIVE_SECONDS_IN_MS = 5000;
-/**
- * What action to take when someone clicks the right-click menu option.
- *  Here it takes the url of the right-clicked image and the current tabId
- *  and forwards it to the imageClassifier's analyzeImage method.
- */
-function clickMenuCallback(info, tab) {
-  imageClassifier.analyzeImage(info.srcUrl, tab.id);
+
+function clickMenuCallback(_, tab) {
+  chrome.tabs.executeScript(tab.id, {
+    code: `var images = []
+           for (var i = 0; i < document.images.length; i++) {
+            images.push(document.images[i].src)
+           }
+
+           chrome.runtime.sendMessage({ method: "downloadImages", images: images })` });
 }
 
-/**
- * Adds a right-click menu option to trigger classifying the image.
- * The menu option should only appear when right-clicking an image.
- */
 chrome.contextMenus.create({
-  title: 'Classify image with TensorFlow.js ',
-  contexts: ['image'],
-  onclick: clickMenuCallback
+  title: "make page shitty",
+  contexts: ["page"],
+  onclick: clickMenuCallback,
+});
+
+chrome.runtime.onMessage.addListener((message) => {
+  if (message.method == "downloadImages") {
+    const allImages = []
+    message.images.forEach(function (v) {
+      allImages.push(v);
+    });
+    allImages.forEach(image => imageClassifier.analyzeImage(image))
```

Comment:
could this not just be `message.images.forEach(xxxx)`

Reply:
brain no work good